.gitignore

```diff
@@ -12,7 +12,9 @@
 .gopath/
 autogen/
 bundles/
-docker/docker
+binary/
```

Comment:
I build locally to binary\ after the split, rather than having my path set to cmd/docker and cmd/dockerd. Only reason why here.